csrankings-g.csv

```diff
@@ -444,7 +444,7 @@ Gianluca Pollastri,University College Dublin,https://people.ucd.ie/gianluca.poll
 Gianluca Stringhini,Boston University,https://seclab.bu.edu/people/gianluca,fM-s2vQAAAAJ
 Gianluigi Ferrari,University of Pisa,http://www.di.unipi.it/~giangi,F2bbBksAAAAJ
 Gianna Del Corso,University of Pisa,http://www.di.unipi.it/~delcorso,AGDMVeIAAAAJ
-Gianni Antichi,Queen Mary University of London,http://eecs.qmul.ac.uk/profiles/antichigianni.html,3VedTqcAAAAJ
+Gianni Antichi,Politecnico di Milano,https://www.deib.polimi.it/eng/people/details/1794222,3VedTqcAAAAJ

```

Comment:
Page lists two affiliations. Is the Milan affiliation >= 75%?

Reply:
Yes. that is correct.